Ginger/GingerCoreCommon/UIElement/ElementInfo.cs

```diff
@@ -55,6 +55,8 @@ public class ElementInfo : RepositoryItemBase
         [IsSerializedForLocalRepository]
         public bool Mandatory { get; set; }
 
+        public bool IsAutoLearned { get; set; }
+
```

Comment:
Need to add "IsAutoLearned" also to ElementInfo Locators and make it true when needed